src/openms/source/ANALYSIS/QUANTITATION/PeptideAndProteinQuant.cpp

```diff
@@ -429,6 +429,11 @@ namespace OpenMS
 
     if (method == "iBAQ")
     {
+      if (proteins.getHits().empty())
+      {
+        OPENMS_LOG_ERROR << "No protein matches found, cannot calculate iBAQ!";
```

Comment:
do we need a hard error? e.g., exception
will probably not make sense to continue here

Reply:
or do we get a meaningful but empty output? then this would be fine I guess

Reply:
The output will be like setting top:N to 0 and top:aggregate to sum. The results won't be divided by the number of theoretical peptides.

Reply:
I guess we can either warn the user, that this happened or stop the execution entirely.

Reply:
if the results don't make sense (e.g., are not iBAQ values) then we should fail. You will not read warnings in a log file when run on 1000 files